### ISSUE TITLE
New 0.3.4 release not published for at least iosarm64 targets

### ISSUE THREAD

**Issue by skolson** [ionspin/kotlin-multiplatform-bignum#213]
Have a KMM build that works fine using 0.3.3. But changing the dependency to 0.3.4 causes gradle sync to fail with this error:
```
   > Could not find com.ionspin.kotlin:bignum-iosarm64:0.3.4.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/ionspin/kotlin/bignum-iosarm64/0.3.4/bignum-iosarm64-0.3.4.pom
       - https://plugins.gradle.org/m2/com/ionspin/kotlin/bignum-iosarm64/0.3.4/bignum-iosarm64-0.3.4.pom
       - https://repo.maven.apache.org/maven2/com/ionspin/kotlin/bignum-iosarm64/0.3.4/bignum-iosarm64-0.3.4.pom
     Required by:
         project :kmp-sc > com.ionspin.kotlin:bignum:0.3.4

```
I'm guessing that the IOS builds didn't get published yet? Or had some publish error? I use iosArm64, iosX64, and macosX64, this is just the first I hit. Haven't looked to see how many other targets are missing in 0.3.4

**Comment by ionspin** [comment]
Hey Steven, thanks for reporting, the reason is: 
:man_facepalming: Mac built targets were published with 0.3.5 instead of 0.3.4 
I should be able to publish them as 0.3.4, but I'll have to skip 0.3.5 altogether. I'll try to get to that later in the day.

**Comment by ionspin** [comment]
0.3.4 Mac built is published, everything should be working properly now
